assets/training/distillation/components/data_generation_file_selector/spec.yaml

```diff
@@ -0,0 +1,54 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+name: oss_distillation_data_generation_file_selector
+version: 0.0.1
+type: command
+
+is_deterministic: True
+tags:
+  codegenBy: dsl.condition_output 
+
+display_name: OSS Distillation Fine-Tuning Input File Selector
+description: Component to select the Batch Scoring Selector based on the task type
+
+environment: azureml://registries/azureml/environments/model-evaluation/labels/latest
+
+inputs:
+  generated_batch_train_file_path:
```

Comment:
same comment for file selector. Should we maintain below i/o schema? This might simplify the schema and we can use this across several use-cases

1. input_file_1 
2. input_file_2
3. condition 

Output: 
1. output_file_1

Reply:
@ayush-ms - As discussed in the PR review, closing this conversation

Reply:
@visahan-24 , you have tagged wrong person

Reply:
@novaturient95  As discussed in the PR review, closing this conversation